### DISCUSSION TITLE
Question about "Vercel Projects Connected per Git Repository"

### DISCUSSION THREAD

**Discussion by bdlowery** [vercel/community#786] [Help] [answered]
For the pro plan it says the maximum is 10.

Does this mean I can only have 10 projects connected to my pro account, or does it just mean if I have 'test-repo-1' I can only make 10 distinct projects with 'test-repo-1'.

**Comment by amyegan** [comment, marked as answer]
Thanks for reaching out @bdlowery! It means you can only make 10 distinct projects from the same repo. You can read more here: https://vercel.com/blog/monorepos#creating-projects-from-a-monorepo
